ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/clientlibs/editor/js/editDialog.js

```diff
@@ -70,7 +83,8 @@
                 }
                 editFormatElement.value="{"+editFormat+"}";
             }
-
+            exclusiveMaximum.value = excludeMaxCheck.checked ? maximum.value : null;
```

Comment:
when excludeMaxCheck.checked is true and we set the exclusiveMaximum to the maximum value, shouldn't maximum value be also set to  null?

Reply:
I followed the `exporter-numberinput.json` of the number input test written. It had both max and exclusiveMax, min and exclusiveMin in the json schema, so I assumed having both of them must be a part of the requirement. Could you confirm if it is so?